tap_exacttarget/endpoints/events.py

```diff
@@ -28,6 +28,10 @@ class EventDataAccessObject(DataAccessObject):
             'type': ['null', 'string'],
             'description': 'The type of tracking event',
         },
+        'URL': {
+            'type': ['null','string'],
+            'description': 'URL that was clicked.'
+        },
```

Comment:
Assuming you run this tap yourself, when you add this URL to the schema and run the tap, do you see this new data being emitted? 

If not, I suspect that we need to add `'URL'` to the `KEY_PROPERTIES` list on line 39.

Reply:
Yes. I tested this by adding `URL` to the schema, and it runs without `URL` being a `KEY_PROPERTY`.

Reply:
@luandy64 do I need to add `URL` to `KEY_PROPERTY`?